app/helpers/guide_helper.rb

```diff
@@ -36,10 +36,12 @@ def guide_community_options_for_select
 
   def topic_section_options_for_select
     Topic.includes(:topic_sections).map do |topic|
+      relative_path = topic.path.gsub("/service-manual", "")
```

Comment:
Or, just use `sub` rather than `gsub` might be OK if we're sure topic.path will _always_ start with `/service-manual`.